tstest/integration/vms/udp_tester.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build ignore
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"os"
+)
+
+var (
+	client = flag.String("client", "", "host:port to connect to for sending UDP")
+	server = flag.String("server", "", "host:port to bind to for receiving UDP")
+)
+
+func main() {
```

Comment:
`nc -u` isn't portable enough?

Reply:
that implies that netcat is on all of the base images and the `netcat` package in the repos is the same netcat. This is not the case and I don't want to have to pick apart the differences if I can avoid it.

Reply:
I'll add a comment though